Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/Process.java

```diff
@@ -62,6 +62,10 @@ public class Process extends BaseTemplateBean {
     @JoinColumn(name = "docket_id", foreignKey = @ForeignKey(name = "FK_process_docket_id"))
     private Docket docket;
 
+    @ManyToOne
```

Comment:
Please add corresponding `@OneToMany` relationship in `OCRWorkflow`.

Reply:
We don't have such a corresponding relationship for similar examples, for example Ruleset or Docket and we don't need this relationship atm. If the relationship is needed then we should add it later and provide an appropriate migration script. I want to customize the data structure only for the cases which are really needed.

@solth What do you mean?